spec/duration.html

```diff
@@ -1714,41 +1714,53 @@ <h1>
           1. Set _months_ and _weeks_ to 0.
         1. Else if _unit_ is *"month"*, then
           1. Assert: CalendarMethodsRecordHasLookedUp(_calendarRec_, ~dateAdd~) is *true*.
+          1. Assert: CalendarMethodsRecordHasLookedUp(_calendarRec_, ~dateUntil~) is *true*.
```

Comment:
Suggestion: move these and their repeats to a separate clause earlier that 'if _unit_ is one of ...' to avoid more assertions and making this text longer?